drivers/clock_control/stm32_ll_clock.c

```diff
@@ -375,6 +423,24 @@ static int stm32_clock_control_init(struct device *dev)
 
 #endif /* CONFIG_CLOCK_STM32_SYSCLK_SRC_... */
 
+#ifndef CONFIG_CLOCK_STM32_MCO1_SRC_NOCLOCK
+#ifndef CONFIG_CLOCK_STM32_MCO1_DIV
+	LL_RCC_ConfigMCO(LL_RCC_MCO1SOURCE);
```

Comment:
Please add a comment that MCO does not activate requested source and on matching source activation should be checked